apps/web/lib/teams/getServerSideProps.tsx

```diff
@@ -1,25 +0,0 @@
-import type { GetServerSidePropsContext } from "next";
-
-import { getServerSession } from "@calcom/features/auth/lib/getServerSession";
-
-import { ssrInit } from "@server/lib/ssr";
-
-export const getServerSideProps = async (context: GetServerSidePropsContext) => {
-  const ssr = await ssrInit(context);
-  await ssr.viewer.me.prefetch();
```

Comment:
these are already done in layout file in app router

https://github.com/calcom/cal.com/blob/main/apps/web/app/(use-page-wrapper)/layout.tsx#L13-L16